### ISSUE TITLE
Make `herman-add` more robust (lenient on first argument singular/plural).

### ISSUE THREAD

**Issue by adrifolio** [oddbird/sassdoc-theme-herman#344]
Hey guys I'm trying to use Heman for generating the Living Style Guide of a project I'm working on and I'm having problems displaying the font specimens.

I'm currently getting this error: 
>Sassjson file is missing font "demo-cdn" data. Did you forget to `@include herman-add()` for this font?

And this is my setup:
In my scss file I'm first importing the herman utilties
```
@import "../../../node_modules/sassdoc-theme-herman/scss/utilities";
```
Then I'm using the example you have in the docs:

```
/// @font demo-cdn (normal, bold, bold italic)
///   <link href="https://fonts.googleapis.com/css?family=Source+Code+Pro:400,700" rel="stylesheet">
$demo-cdn: (
$demo-cdn: (
  'name': 'Source Code Pro',
  'source': 'https://fonts.google.com/specimen/Source+Code+Pro',
  'stack': (
    'Consolas',
    'Menlo',
    'Courier New',
    monospace,
    sans-serif,
  ),
);  

@include herman-add('font', 'demo-cdn', $demo-cdn);
@include herman-export ($demo-cdn);
```
In my `.sassdocrc` I have:

```
theme: herman
verbose: true

herman:
  sass:
    jsonfile: dist/assets/css/main.css
```

And finally this is the generated json I get on `main.css`:

```
/*! json-encode: {"name": "Source Code Pro", "source": "https://fonts.google.com/specimen/Source+Code+Pro", "stack": ["Consolas", "Menlo", "Courier New", "monospace", "sans-serif"]} */
/*# sourceMappingURL=main.css.map */
```

Is the generated json correct? I don't see a reference to `demo-cdn` so I wonder if that is the issue. I appreciate your help!!

**Comment by lkodai** [comment]
Bump. I'm having this issue as well, any guidance from the Herman team would be appreciated.

**Comment by mirisuzanne** [comment]
No, that json does not look right. It should all be nested under a `fonts` object. Looks like you need to change `'font'` to `'fonts'` in `herman-add()` (but we might want to make that more robust.

```scss
@include herman-add('fonts', 'demo-cdn', $demo-cdn);
```

**Comment by mirisuzanne** [comment]
Looks like this was fixed by e554bdacc31d8102a4f549ea06bf9294ba16583a
